src/test/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/BlobProcessorTest.java

```diff
@@ -46,33 +63,19 @@
     @Mock BlobLeaseClient blobLeaseClient;
     @Mock EnvelopeRepository envelopeRepo;
     @Mock BlobSignatureVerifier signatureVerifier;
-
-    BlobProcessor blobProcessor;
-
-    @BeforeEach
-    void setUp() {
-        this.blobProcessor = new BlobProcessor(
```

Comment:
Creating a new blob processor on the fly, as it uses one of its dependencies (differently mocked, depending on test) in constructor.